src/wp-admin/about.php

```diff
@@ -56,10 +54,8 @@
 			</div>
 		</div>
 
-		<hr />
-
 		<div class="about__section has-2-columns">
-			<div class="column about__image is-vertically-aligned-center">
+			<div class="column about__image is-vertically-aligned-top">
 				<img src="https://make.wordpress.org/core/files/2022/05/about-60-writing-improvements.png" alt="" />
```

Comment:
Excuse my ignorance, but when do these files typically get uploaded to the CDN?

Reply:
Thanks for the reminder, that needs to happen today.